src/Kunstmaan/NodeBundle/AdminList/NodeAdminListConfigurator.php

```diff
@@ -46,6 +48,8 @@ class NodeAdminListConfigurator extends AbstractDoctrineORMAdminListConfigurator
      */
     protected $authorizationChecker;
 
+    protected ?Node $node = null;
```

Comment:
I would make this private for BC reasons, we can still expose it if needed in the future